tests/functional/prometheous/test_prometheus_validations_file.py

```diff
@@ -4,19 +4,16 @@
 except ImportError:
     # py2
     import json
+
 import time
 
 import ddt
+import pytest
```

Comment:
All imports should be in ascending order, importing pytest is disturbing the order.